src/store/PriceLineStore.js

```diff
@@ -60,6 +65,10 @@ export default class PriceLineStore {
         return display;
     }
 
+    @computed get yAxiswidth() {
```

Comment:
@balakrishna-binary While `yAxiswidth` is not observable, there is not any need to use `@computed` decorator.

Reply:
Got it. Thanks!